ovos-i2csound

```diff
@@ -159,7 +189,11 @@ main() {
     if [[ ${detection_results[TAS5806]} == true ]] ; then
         echo "Installing and configuring SJ-201 HAT"
         # Initializing XMOS xvf3510
-        dtoverlay xvf3510
```

Comment:
This breaks compat with the overlay used for kernels 5.10 to 6.6. Is this intentional or should there be a check here for which overlay exists?

Reply:
Not intentional. Let me add a check to that.